<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>807</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>80</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="stopButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>451</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5"/>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>150</y>
      <width>241</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>120</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Experiment Notes</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>581</width>
      <height>351</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="gentab">
     <attribute name="title">
      <string>General</string>
     </attribute>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>MCCDAQ - Universal Library</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Hardware Family</string>
      </property>
     </widget>
     <widget class="QLabel" name="output_display">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>251</width>
        <height>16</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>No Folder Selected</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Folder</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>File Output Location</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>140</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Session Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputsession">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>160</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>null</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>180</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Files to be Generated</string>
      </property>
     </widget>
     <widget class="QLabel" name="output_file1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>200</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>null_data.bin</string>
      </property>
     </widget>
     <widget class="QLabel" name="output_file2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>220</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>null_info.txt</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Individual Channel Recording Frequency</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>80</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Please be aware of your device's maximum per-channel recording frequency.</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>30</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLineEdit" name="inputrecordingfreq">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>Hz</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="output_recfreq">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>60</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Currently: 0 Hz</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>10</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Misc Options</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="savesettings">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>30</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Save Settings</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>200</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>HighThresh</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="highthresh">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>V</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>260</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TRIGGER INFORMATION</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>280</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Files in select file output location will automatically increment upon each new trigger.</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>280</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Datablock Duration</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>190</y>
        <width>151</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>After Trigger Fires:</string>
      </property>
      <widget class="QCheckBox" name="trigger_multi">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>131</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Reset for New Trigger</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="trigger_datablock">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>111</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Record Datablock</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="trigger_keeprecording">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>111</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Keep Recording</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>150</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TRIGGER OPTIONS</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="usetrigger">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>170</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Use Trigger</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>160</y>
        <width>81</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>(Optional)</string>
      </property>
      <widget class="QCheckBox" name="setting_overwrite">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Overwirte</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="setting_increment">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Increment</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>230</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>LowThresh </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lowthresh">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>V</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>300</y>
        <width>75</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLineEdit" name="trigger_duration">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>(s)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QCheckBox" name="onetext">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>170</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Output Only One Textfile</string>
      </property>
     </widget>
     <zorder>layoutWidget</zorder>
     <zorder>layoutWidget</zorder>
     <zorder>comboBox</zorder>
     <zorder>label</zorder>
     <zorder>output_display</zorder>
     <zorder>pushButton</zorder>
     <zorder>label_19</zorder>
     <zorder>label_20</zorder>
     <zorder>inputsession</zorder>
     <zorder>label_21</zorder>
     <zorder>output_file1</zorder>
     <zorder>output_file2</zorder>
     <zorder>label_22</zorder>
     <zorder>label_23</zorder>
     <zorder>layoutWidget_2</zorder>
     <zorder>output_recfreq</zorder>
     <zorder>label_25</zorder>
     <zorder>savesettings</zorder>
     <zorder>layoutWidget_3</zorder>
     <zorder>label_18</zorder>
     <zorder>label_28</zorder>
     <zorder>label_29</zorder>
     <zorder>groupBox</zorder>
     <zorder>label_30</zorder>
     <zorder>usetrigger</zorder>
     <zorder>groupBox_2</zorder>
     <zorder>onetext</zorder>
    </widget>
    <widget class="QWidget" name="chantab">
     <attribute name="title">
      <string>Channels</string>
     </attribute>
     <widget class="QwtCounter" name="chancounter">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>82</width>
        <height>20</height>
       </rect>
      </property>
      <property name="value">
       <double>0.000000000000000</double>
      </property>
      <property name="minimum">
       <double>0.000000000000000</double>
      </property>
      <property name="maximum">
       <double>8.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>1.000000000000000</double>
      </property>
      <property name="numButtons">
       <number>1</number>
      </property>
      <property name="stepButton2">
       <number>3</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Number of Channels</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Channel Name</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>242</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="chan1layout">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Analog Input 0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="chan1name"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>242</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="chan2layout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Analog Input 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="chan2name"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>140</y>
        <width>242</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="chan3layout">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Analog Input 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="chan3name"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>170</y>
        <width>242</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="chan4layout">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Analog Input 3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="chan4name"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>200</y>
        <width>242</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="chan5layout">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Analog Input 4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="chan5name"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>230</y>
        <width>242</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="chan6layout">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Analog Input 5</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="chan6name"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>260</y>
        <width>242</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="chan7layout">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Analog Input 6</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="chan7name"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>290</y>
        <width>242</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="chan8layout">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Analog Input 7</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="chan8name"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Plot</string>
     </attribute>
     <widget class="QLabel" name="output_recfreq_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Currently: 0 Hz</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Plot Frequency</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="inputrecordingfreq_2">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_26">
         <property name="text">
          <string>Hz</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>40</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm Settings</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>875</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Images/save.png</normaloff>:/Images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Images/new.png</normaloff>:/Images/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Images/open.png</normaloff>:/Images/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtCounter</class>
   <extends>QWidget</extends>
   <header>qwt_counter.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
